src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/PacketHandle.netcore.unix.cs

```diff
@@ -2,20 +2,23 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-#if NET
+// @TODO: Introduce polymorphism to remove need for this level of indirection
+// @TODO: Also, why do we have any other type besides managed defined here?
```

Comment:
PacketHandles are used to transport either native or managed packets around TdsParserStateObject.

Reply:
I know we discussed this before, but I'm still not convinced ... so I left these comments to consider it further.
But considering on unix, we never have native packets, I don't know why we need to have an object with both types defined.